src/frontend/src/pages/CalendarPage/DesignReviewDetailPage/FinalizeDesignReviewModal.tsx

```diff
@@ -0,0 +1,108 @@
+import {
+  Box,
+  FormControl,
+  Grid,
+  InputLabel,
+  MenuItem,
+  Select,
+  TextField,
+  ToggleButton,
+  ToggleButtonGroup,
+  Typography
+} from '@mui/material';
+import NERModal from '../../../components/NERModal';
+import { useState } from 'react';
+
+interface FinalizeDesignReviewProps {
+  open: boolean;
+  setOpen: (val: boolean) => void;
+  onSubmit?: () => void;
+}
+
+const FinalizeDesignReviewModal = ({ open, setOpen, onSubmit }: FinalizeDesignReviewProps) => {
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const names = ['jdel', 'dehkdh', 'dehjkdhk', 'dsahksdh', 'dsajkldsaj'];
+  const locations = ['The Bay', 'Zoom'];
+  const [docTemplate, setDocTemplate] = useState('');
+  const [meetingType, setMeetingType] = useState<string[]>([]);
+  const [location, setLocation] = useState('');
+
+  const handleMeetingTypeChange = (_event: any, newMeetingType: string[]) => {
+    setMeetingType(newMeetingType);
+  };
+
+  const handleLocationChange = (event: any) => {
+    setLocation(event.target.value);
+  };
+
+  return (
+    <NERModal open={open} title={''} onHide={handleClose} onSubmit={onSubmit} submitText="Finalize">
+      <Grid>
+        <Grid>
+          <Grid style={{ display: 'flex', marginBottom: 20 }}>
+            <Typography style={{ textDecoration: 'underline', fontSize: '1.2em', marginRight: 50 }}>
+              Add Doc Template:
+            </Typography>
+            <TextField
+              label="Type here"
+              variant="outlined"
+              value={docTemplate}
+              onChange={(e) => setDocTemplate(e.target.value)}
+              style={{ backgroundColor: 'grey', borderRadius: 10 }}
+              size="small"
+            />
+          </Grid>
+          <Grid style={{ display: 'flex', marginBottom: 20 }}>
+            <Typography style={{ textDecoration: 'underline', fontSize: '1.2em', marginRight: 95 }}>
+              Meeting Type:
+            </Typography>
+            <ToggleButtonGroup color="primary" value={meetingType} onChange={handleMeetingTypeChange}>
+              <ToggleButton value="virtual">Virtual</ToggleButton>
+              <ToggleButton value="inPerson">In-person</ToggleButton>
+            </ToggleButtonGroup>
+          </Grid>
+          <Grid style={{ display: 'flex', alignItems: 'center', marginBottom: 50 }}>
+            <Typography style={{ textDecoration: 'underline', marginBottom: 10, fontSize: '1.2em', marginRight: 135 }}>
+              Location:
+            </Typography>
+            <FormControl style={{ backgroundColor: 'grey', width: '45%', borderRadius: 5 }} fullWidth>
+              <InputLabel>Location</InputLabel>
+              <Select value={location} onChange={handleLocationChange} variant="outlined">
```

Comment:
Location is just a string, so this should be a text input. This is for physical locations, so zoom isn't an option. Its usually gonna be in the bay so if we could do like "The Bay" or a custom input then we should, but if not we should just do a text input